pages/price-feeds/how-pyth-works/schedule-format.mdx

```diff
@@ -0,0 +1,52 @@
+# Schedule Format
+
+This document describes the format for the `schedule` field in Pyth product metadata.
+The `schedule` field defines a product's market hours, including its timezone, weekly schedule, and any holiday exceptions.
+It instructs the `pyth-agent` when to stop publishing outside the specified hours.
+This field is particularly useful for:
+
+- FX
+- Metals
+- Stocks
+
+## Format
+
+```plain
+Timezone;WeeklySchedule;Holidays
+```
+
+**If `schedule` is omitted for a symbol, pyth-agent will revert to the deprecated `weekly_schedule` field.
+If `weekly_schedule` is also undefined, pyth-agent will default to 24/7 publishing, which was the standard behavior prior to this feature.**
+
+**Detailed Definitions**
+
+- `Timezone` - A human-readable tz database TZ identifier of the market’s local timezone, such as `America/New_York`.
+  **Full list of identifiers can be found [here](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)**.
+  Daylight-saving time is automatically handled by the pyth-agent.
```

Comment:
We can add Daylight-saving as a callout.

Reply:
Good idea. Done.